### DISCUSSION TITLE
Error message in Orange Pi 5 Plus

### DISCUSSION THREAD

**Discussion by SilverDesert** [Joshua-Riek/ubuntu-rockchip#240] [General]
I am getting this error message in the Orange Pi 5 Plus. It was noted in version1.18 and is also noted in version 1.19. Any thoughts?
![20230701_232343](https://github.com/Joshua-Riek/ubuntu-rockchip/assets/123965592/fceb1811-ae10-476b-a057-009cf12e2979)

**Comment by Joshua-Riek** [comment]
I have seen this before, and it does not seem to be a problem. My patch to silence this message must have been lost when the kernel was updated to 5.10.160. I'm working on a few kernel patches at the moment and will add the below patch. 

```diff
From 49e8e4743cc72a9bfb443468d3d50a4c2414f0b3 Mon Sep 17 00:00:00 2001
From: Joshua Riek <jjriek@verizon.net>
Date: Sat, 1 Jul 2023 17:50:19 -0400
Subject: [PATCH] drm/rockchip: dw-dp: suppress "timeout waiting for AUX reply" spam

---
 drivers/gpu/drm/rockchip/dw-dp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/dw-dp.c b/drivers/gpu/drm/rockchip/dw-dp.c
index edef80f3e723..004ca1db01f5 100644
--- a/drivers/gpu/drm/rockchip/dw-dp.c
+++ b/drivers/gpu/drm/rockchip/dw-dp.c
@@ -2686,7 +2686,7 @@ static ssize_t dw_dp_aux_transfer(struct drm_dp_aux *aux,
 
 	status = wait_for_completion_timeout(&dp->complete, timeout);
 	if (!status) {
-		dev_err(dp->dev, "timeout waiting for AUX reply\n");
+		//dev_err(dp->dev, "timeout waiting for AUX reply\n");
 		return -ETIMEDOUT;
 	}
 
-- 
2.25.1
```

**Comment by SilverDesert** [comment]
Thanks.
There is also another error message. Just see the 2nd line with ubuntu login...
![20230630_162022](https://github.com/Joshua-Riek/ubuntu-rockchip/assets/123965592/5a5d3d75-0b87-4f8e-a1d6-911d94a20943)

**Reply by Joshua-Riek** [reply]
That one is normal. It's related to the HDMI display panes. The vop2 display driver has a lot of weird bugs and quirks. I have been able to fix most issues, but a few remain.

**Reply by SilverDesert** [reply]
Ok, thanks.
